src/hotspot/share/gc/parallel/psScavenge.cpp

```diff
@@ -521,31 +521,56 @@ void PSScavenge::clean_up_failed_promotion() {
 }
 
 bool PSScavenge::should_attempt_scavenge() {
-  ParallelScavengeHeap* heap = ParallelScavengeHeap::heap();
+  const static bool SHOULD_RUN_YOUNG_GC = true;
+  const static bool SHOULD_RUN_FULL_GC = false;
```

Comment:
These two constants do not seem to be `static`.

Reply:
It probably doesn't make any difference. I can drop `static` if you prefer.

Reply:
Generally, the `static` variables in a method are used to save the state of the previous invocation and then used by the following invocations. But we only need a named constant here. So I think it is better to remove the keyword `static`.